aten/src/ATen/native/mkldnn/xpu/Blas.cpp

```diff
@@ -93,9 +94,13 @@ Tensor& addmm_out(
     }
   } else {
     if (alpha.to<float>() == 1.f && beta_ == 1.f) {
-      bias = self;
+      bias = is_inplace ? self.clone() : self;
```

Comment:
It would be better to add some comments to elaborate on why the clone is required here.

Reply:
Sure, the comments is added here